scss/responsive/_large.scss

```diff
@@ -24,6 +24,11 @@
         font-size: 1.8em;
       }
     }
+    .oo-info-panel-cast {
+      p {
+        font-size: 1.8em;
+      }
+    }
```

Comment:
1.8em is a common size for "oo-info-panel-cast" and "oo-state-screen-title" classes?
Could we move it into sass-constant? :-)

Reply:
You're right, I'll move it to a constant